libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -92,6 +92,15 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("TER_MARGIN",  12, AC_WPNav, _terrain_margin, 10.0),
 
+    // @Param: ACCEL_C
+    // @DisplayName: Waypoint Cornering Acceleration
+    // @Description: Defines the maximum cornering acceleration in cm/s/s used during missions, zero uses max lean angle.
```

Comment:
We should remove the "Defines the" part because all parameters define something or configure something.  This is not a blocker though.